CMake/itkExternal_Eigen3.cmake

```diff
@@ -73,9 +73,8 @@ else()
   endif()
   install(
     CODE "execute_process(
-    COMMAND
-    ${CMAKE_COMMAND} --build . --config Release --target install
-    WORKING_DIRECTORY ${_eigen3_build_dir}
+    COMMAND \${CMAKE_COMMAND} --build . --config Release --target install
+    WORKING_DIRECTORY \"${_eigen3_build_dir}\"
```

Comment:
Isn't enough with the `"` in `_eigen3_build_dir`?
```cmake
  set(_eigen3_source_dir "${ITK_SOURCE_DIR}/Modules/ThirdParty/Eigen3/src/itkeigen")
  set(_eigen3_build_dir "${ITK_BINARY_DIR}/ITKInternalEigen3-build")
```

Reply:
No, the quotes on those `set` lines are consumed by the cmake language parser and do not become part of the value.